test/test_helper.exs

```diff
@@ -4,6 +4,8 @@
 
 {:ok, _} = Application.ensure_all_started(:wallaby)
 
+Mimic.copy(GSearcher.Repo)
```

Comment:
Could you help me to understand why do we need this @bterone

Reply:
It's similar to `Mox.defmock(MyApp.MockWeatherAPI, for: MyApp.WeatherAPI)` where it creates a duplicate of the `Module` we give so that we can `stub` or `expect` methods from this module. 😄

Reply:
@bterone Please correct me if I'm wrong but I don't see any Mimic been using in this template, seems we don't need to stub anything yet?

Reply:
Oh, it's here `https://github.com/bterone/g-searcher/blob/feature/oauth-signup/test/g_searcher_web/controllers/authentication_controller_test.exs#L49`